build-arm64.yaml

```diff
@@ -0,0 +1,59 @@
+directories:
+  output: dist
+  buildResources: build
+  app: app_compiled
+appId: org.poooi.poi
+asar: true
+npmRebuild: false
+mac:
+  publish: []
+  icon: assets/icons/poi.icns
+  category: public.app-category.games
+  provisioningProfile: poi.provisionprofile
+  target:
+    - target: default
+      arch:
+        - arm64
+dmg:
+  contents:
+    - x: 410
+      'y': 220
+      type: link
+      path: /Applications
+    - x: 130
+      'y': 220
+      type: file
+win:
+  publish: []
+  icon: assets/icons/poi.ico
+  target:
+    - target: nsis
+      arch:
+        - arm64
+    - target: 7z
+      arch:
+        - arm64
+nsis:
+  artifactName: 'poi-setup-${version}-arm64.${ext}'
```

Comment:
This adds `arm64` suffix to the NSIS file name. But `electron-builder` also creates a `latest.yml` to use with `electron-updater`. The name is `latest.yml` for both `x64` and `arm64` so they end up overwriting each other. Any solution?

Reply:
I managed to patch this. You can apply this patch to poi as well: https://github.com/webcatalog/webcatalog-app/pull/1253/files